res/values-b+es+r419/strings.xml

```diff
@@ -1328,21 +1328,21 @@ Subir la misma skin NO aprobada varias veces resultará en más sanciones o en u
     <string name="Custom_Skin_Reviewed_Notification">¡Tu skin personalizada ha sido revisada!</string>
     <string name="You_Have_Mail_">Tienes un correo de %s!</string>
     <string name="Clan_Request_Reviewed_Notification">¡Tu solicitud para unirte al Clan %s ha sido revisada!</string>
-    <string name="CRAZY_SPLIT">CRAZY SPLIT 32x</string>
+    <string name="CRAZY_SPLIT">SEPARARSE LOCO 32x</string>
```

Comment:
This isn't supposed to be changed, otherwise the typeface won't work well. Please leave them as they were, especially because people are already used to that name.